README.md

```diff
@@ -502,14 +502,30 @@ npx prisma migrate dev --name <migration-name>
 
 This will create a new migration file in [/prisma/migrations](prisma/migrations).
 
-Currently these migrations are not automatically run in our CI/CD pipeline, so it has to be done manually:
-
-1. Run the [cloud sql auth proxy locally](#google-cloud--cloud-sql-auth-proxy-setup)
-2. Update your environment variables to point to the production database.
-3. Run `npx prisma migrate deploy` to apply the migration to the production database.
```

Comment:
No need to keep this moving forward right?

Reply:
Yay 🎉